packages/table/src/index.ts

```diff
@@ -4,8 +4,6 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import "es6-shim";
```

Comment:
you need to add polyfills to docs-app and landing-app. I suggest core-js submodules.

Reply:
truth. we're using `es6-shim` in devDeps in every package for tests. should I switch wholesale over to `core-js`?

Reply:
yeah, switch wholesale